install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1772,7 +1772,11 @@ loadBalancer:
   # mode: snat
 
   # -- acceleration is the option to accelerate service handling via XDP
-  # e.g. native, disabled
+  # disabled - do not use XDP
+  # native - XDP BPF program is run directly out of the networking driver’s
+  #   early receive path.
+  # best-effort - Use native mode XDP acceleration on devices that support
+  #   it.
```

I see you updated both `.../values.yaml` and `.../values.yaml.tmpl`, but you also need to run `make -C install/kubernetes` to propagate the change to `install/kubernetes/cilium/README.md`.

When you do so, you should observe that the formatting you're using here (the list) translates rather badly in the `README.md`, where the comment lines are joined together. Can you please improve it? For example:

```suggestion
  # Applicable values can be: disabled (do not use XDP), native (XDP BPF
  # program is run directly out of the networking driver's early receive path),
  # or best-effort (use native mode XDP acceleration on devices that support it).
```

---

I thought I had run `make -C install/kubernetes` in order to generate values.yaml but I didn't see it making any changes to install/kubernetes/cilium/README.md. I'll do this again and dig a little bit.

---

Ok - figured out the issue here.

Since `acceleration` was cancelled out it was never showing up in the helm chart's README.md file since it wasn't registering as being a value that could be set.

For now I have uncommented it in `values.yaml.tmpl` setting it to `disabled` as the default and ran `make -C install/kubernetes`.

it's now documented in the README.

Does this look good @qmonnet?

---

OK, sorry I didn't realise the `acceleration` was simply left out. I didn't mean you had to uncomment it, but that's probably fine. And yes, the reformatted comment looks better to me, thank you!

---

Ah, if you keep the uncommented `acceleration`, then you need to run `make -C Documentation update-helm-values` as well and commit the changes. This is what the integration tests complain about.

---

Ok - I think we are good here.

---

Also, thank you for all of the help with this!